docs/installation/on_premise/minimal_conf.md

```diff
@@ -0,0 +1,163 @@
+
+=== "maven"
+    ``` xml
+    <plugin>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-maven-plugin</artifactId>
+        <version>2.7.5</version>
+        <configuration>
+            <executable>true</executable>
+            <layout>ZIP</layout>
+            <mainClass>com.chutneytesting.ServerBootstrap</mainClass>
+            <finalName>chutney-${project.artifactId}-${chutney.version}</finalName>
+        </configuration>
+        <executions>
+            <execution>
+                <goals>
+                    <goal>repackage</goal>
+                </goals>
+            </execution>
+        </executions>
+    </plugin>
+    ```
+
+Declare a BOM dependency on Chutney parent.
+
+=== "maven"
+    ``` xml
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>com.chutneytesting</groupId>
+                <artifactId>chutney-parent</artifactId>
+                <version>${chutney.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+    ```
+
+Add Chutney server and UI as dependencies.
+
+=== "maven"
+    ``` xml
+    <dependency>
+        <groupId>com.chutneytesting</groupId>
+        <artifactId>server</artifactId>
+        <scope>runtime</scope>
+    </dependency>
+    <dependency>
+        <groupId>com.chutneytesting</groupId>
+        <artifactId>ui</artifactId>
+        <scope>runtime</scope>
+    </dependency>
+    ```
+
+Then, add dependency for your chosen database.
+
+=== "maven"
+    ``` xml
+    <dependency> <!-- (1) -->
+        <groupId>com.h2database</groupId>
+        <artifactId>h2</artifactId>
+    </dependency>
+    <dependency> <!-- (2) -->
+        <groupId>org.postgresql</groupId>
+        <artifactId>postgresql</artifactId>
+    </dependency>
+    ```
+
+    1. If you want to use H2 as Chutney main database
+    2. If you want to use PostgreSQL as Chutney main database
+
+Also, you should add any dependencies you would need to run your scenarios.  
+This may depend on the underlying Chutney actions you are using.
+
+=== "maven"
+    ``` xml
+    <dependency> <!-- (1) -->
+        <groupId>com.oracle</groupId>
+        <artifactId>ojdbc6</artifactId>
+        <version>x.x.x</version>
+        <scope>runtime</scope>
+    </dependency>
+    <dependency> <!-- (2) -->
+        <groupId>weblogic</groupId>
+        <artifactId>wlthinclient</artifactId>
+        <version>x.x.x</version>
+        <scope>runtime</scope>
+    </dependency>
+    ```
+
+    1. Example for using [SQL actions](/documentation/actions/sql) and query an Oracle database
+    2. Example for using [JMS actions](/documentation/actions/jms) with a WebLogic server
+
+Finally, add your own [Actions](/documentation/actions/introduction) and [Functions](/documentation/functions/classpath)
```

Comment:
How to add custom actions and functions is not documented.
May be document it in an other page and link it here ?

Reply:
done